bin/ycsb

```diff
@@ -43,7 +43,8 @@ DATABASES = {
     "nosqldb"      : "com.yahoo.ycsb.db.NoSqlDbClient",
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
     "redis"        : "com.yahoo.ycsb.db.RedisClient", 
-    "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient", 
+    "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient",
+    "couchbase"    : "com.yahoo.ycsb.db.CouchbaseClient"
```

Comment:
Nit picking, but it would be nice to keep these alphabetical. Fine to defer until after this PR is merged.